src/tools/dataset.cpp

```diff
@@ -1708,6 +1689,14 @@ void ustore_graph_export(ustore_graph_export_t* c_ptr) {
 
     if (!c.arena)
         c.arena = arena_t(c.db).member_ptr();
+    auto arena = linked_memory(c.arena, c.options, c.error);
+    return_if_error_m(c.error);
+
+    auto handle_exception = [&](char const* ex) {
+        auto ptr = arena.alloc<char>(strlen(ex), c.error);
```

Comment:
How do you decide to use `std::` prefix or not in this line or beneath for `strlen` and `std::memcpy`?